module-services/service-cellular/DTMFCode.hpp

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+#include <stdint.h>
+#include <string>
+
+class DTMFCode
+{
+  public:
+    explicit DTMFCode(char digitChar);
+    explicit DTMFCode(const char *digitString);
```

Comment:
Please add doxy docks how these work :) ( is `char` ascii char, or integer value - it would be nice to tell here :) )

Reply:
imo that was the problem, that uint32_t has been used to transfer ASCII code instead of built-in type - `char`. Now it's much more explicit that it has to be digit CHAR, not the digit itself, but sure, will add

Reply:
added